rfcs/0003-xcmp-picasso-statemine.md

```diff
@@ -0,0 +1,252 @@
+            type: "sp_runtime:multiaddress:MultiAddress",
+            value: {
+              Id: "0x6d6f646c70792f74727372790000000000000000000000000000000000000000"
+            }
+          },
+          {
+            name: "dest",
+            type: "sp_runtime:multiaddress:MultiAddress",
+            value: {
+              Id: "0x70617261e8030000000000000000000000000000000000000000000000000000"
+            }
+          },
+          {
+            name: "value",
+            type: "compact<U128>",
+            value: "11000000000000"
+          }
+        ]
+      },
+      {
+        call_index: "6300",
+        call_module: "XcmPallet",
+        call_name: "send",
+        params: [
+          {
+            name: "dest",
+            type: "xcm:VersionedMultiLocation",
+            value: {
+              V1: {
+                interior: {
+                  X1: {
+                    Parachain: 1000
+                  }
+                },
+                parents: 0
+              }
+            }
+          },
+          {
+            name: "message",
+            type: "xcm:VersionedXcm",
+            value: {
+              V2: [
+                {
+                  Transact: {
+                    call: "1f00010100020c000400000000070010a5d4e81300000000070010a5d4e800060002286bee5c1802083c01270800003c0027080000e803000000900100",
+                    origin_type: "Superuser",
+                    require_weight_at_most: 1000000000
+                  }
+                }
+              ]
+            }
+          }
+        ]
+      }
+    ]
+  }
+]
+```
+
+Where transact [decodes](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fstatemine-rpc.dwellir.com#/extrinsics/decode) by Statemine into:
+
+```rust
+polkadotXcm
+send(dest, message)
+send
+dest: XcmVersionedMultiLocation
+{
+  V1: {
+    parents: 1
+    interior: Here
+  }
+}
+message: XcmVersionedXcm
+{
+  V2: [
+    {
+      WithdrawAsset: [
+        {
+          id: {
+            Concrete: {
+              parents: 0
+              interior: Here
+            }
+          }
+          fun: {
+            Fungible: 1,000,000,000,000
+          }
+        }
+      ]
+    }
+    {
+      BuyExecution: {
+        fees: {
+          id: {
+            Concrete: {
+              parents: 0
+              interior: Here
+            }
+          }
+          fun: {
+            Fungible: 1,000,000,000,000
+          }
+        }
+        weightLimit: Unlimited
+      }
+    }
+    {
+      Transact: {
+        originType: Native
+        requireWeightAtMost: 1,000,000,000
+        call: {
+          encoded: 0x1802083c01270800003c0027080000e803000000900100
+        }
+      }
+    }
+  ]
+}
+```
+
+Where [Kusama.utility.batchAll](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama-rpc.polkadot.io#/extrinsics) is encoded as `0x1802083c01270800003c0027080000e803000000900100`
+
+## Accept request from Statemine
+
+Accept channel from state mine `0x3c01e8030000`
```

Comment:
Can this fail on Picasso side? if so what would happen?

Reply:
Yes. I guess this is out of scope of proposal how to debug issues. Can discuss in other place.